grype/presenter/cyclonedx/vulnerability_test.go

```diff
@@ -0,0 +1,51 @@
+package cyclonedx
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCvssVersionToMethod(t *testing.T) {
+	testCases := []struct {
+		desc     string
+		input    string
+		expected string
+		errors   bool
+	}{
+		{
+			desc:     "invalid (not float)",
+			input:    "",
+			expected: "",
+			errors:   true,
+		},
+		{
+			desc:     "CVSS v3",
+			input:    "3.1",
+			expected: "CVSSv3",
+			errors:   false,
+		},
+		{
+			desc:     "CVSS v3",
+			input:    "3.1",
+			expected: "CVSSv3",
+			errors:   false,
+		},
+		{
+			desc:     "invalid (no match)",
+			input:    "15.4",
+			expected: "",
+			errors:   true,
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.desc, func(t *testing.T) {
+			actual, err := cvssVersionToMethod(tc.input)
+			if !tc.errors {
+				assert.Equal(t, nil, err)
```

Comment:
nit: this could be swapped for testify's [`assert.NoError(t, err)`](https://pkg.go.dev/github.com/stretchr/testify/assert#NoError)

Reply:
adjusted in https://github.com/anchore/grype/pull/317/commits/528d45eaa973efc85357734adbfa5588dac6dde8